tests/endToEnd.py

```diff
@@ -212,13 +218,15 @@ def testEndToEnd(self):
         #        datarel needs imsim_*.
         ver = 'imsim-2010-12-17-1'
         print "Setting up astrometry_net_data", ver
+        # XXX what is actually used from this setup -- a path in the env?
```

Comment:
I don't think it is that simple because there is a version parameter being passed.